Makefile

```diff
@@ -70,7 +70,7 @@ lint:
 	if [ "$$lint" != "" ]; then exit 1; fi
 
 vet:
-	go tool vet -all -shadow $(shell ls -d */ | grep -v vendor)
+	go tool vet -all -shadow $(shell if go version | grep -v 1.5 | grep -v 1.4 >> /dev/null; then echo -n "-example"; fi) $(shell ls -d */ | grep -v vendor)
```

Comment:
Shouldn't this be just echo "-example". For whatever reason echo is echoing out -n -example, which breaks the vet tool